api/app/config.py

```diff
@@ -143,7 +143,8 @@ class Config(object):
     SMS_CHAR_COUNT_LIMIT = 495
     TEST_MESSAGE_FILENAME = 'Test message'
     ONE_OFF_MESSAGE_FILENAME = 'Report'
-    MAX_VERIFY_CODE_COUNT = 10
+    MAX_VERIFY_CODE_COUNT = 1
```

Comment:
does this get used anywhere anymore?

Reply:
Yeah it's used [here](https://github.com/govau/notify/pull/185/files#diff-92235129d06e5131f2efeb1f4bed2e9bR142)